packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -31,14 +32,20 @@ describe('ui:create:atomic', () => {
       projectName
     );
 
-    await Promise.race([
+    const buildTerminalExitPromise = Promise.race([
       buildTerminal.when('exit').on('process').do().once(),
       buildTerminal
         .when(/Happy hacking!/)
         .on('stdout')
         .do()
         .once(),
     ]);
+
+    await buildTerminal
+      .when(/\(y\)/)
+      .on('stderr')
+      .do(answerPrompt(`y${EOL}`))
+      .until(buildTerminalExitPromise);
```

Comment:
This is only triggered with the local setup because `npx` behave differently in a CI (it warn and install without prompt)